examples/accounts/accountDataFetchers.tsx

```diff
@@ -0,0 +1,13 @@
+import type { AuthorizedParty } from '../../lib/hooks/useAccountSelector';
+import * as data from './authorizedPartiesData.json';
+
+export const getAuthorizedPartiesData = (): AuthorizedParty[] => {
+  // Handle both default export and direct array import
+  // biome-ignore lint/suspicious/noExplicitAny: Need to handle potential default export vs direct import
+  const arrayData = (data as any).default || data;
+  return arrayData as AuthorizedParty[];
+};
+
+export const getFavoriteAccountUuidsData = (): string[] => {
```

Comment:
I don't see that this in use?

Reply:
Yeah, I ended up not using it after all 😅
I'll delete it!